kagiso_auth/forms.py

```diff
@@ -56,11 +56,16 @@ class SignUpForm(forms.Form):
     )
     password = forms.CharField(widget=forms.PasswordInput(), min_length=8)
     confirm_password = forms.CharField(widget=forms.PasswordInput())
-    mobile = forms.CharField(  # TODO: RegexField?
+    mobile = forms.RegexField(
+        regex=r'^\d{10}$',
```

Comment:
Reason for two regex's: HTML 5 client-side validation, and server-side Django validation